src/components/Navigation/NavigationItemTrigger/NavigationItemTrigger.tsx

```diff
@@ -16,8 +17,10 @@ export interface NavigationItemTriggerProps extends ComponentProps<'li'> {
 }
 
 const MobileNavigationItemTrigger = ({ id, label, children }: NavigationItemTriggerProps) => {
+  const { isMenuOpen } = React.useContext(HeaderContext);
```

Comment:
Nitpick: we could have pulled this from the context we are using the the component and passed to the mobile trigger.